src/core/nonbonded_interactions/lj.hpp

```diff
@@ -45,24 +45,24 @@ int lennard_jones_set_params(int part_type_a, int part_type_b, double eps,
 inline void add_lj_pair_force(IA_parameters const *ia_params,
                               Utils::Vector3d const &d, double dist,
                               Utils::Vector3d &force) {
-  if ((dist < ia_params->LJ_cut + ia_params->LJ_offset) &&
-      (dist > ia_params->LJ_min + ia_params->LJ_offset)) {
-    auto const r_off = dist - ia_params->LJ_offset;
-    auto const frac6 = Utils::int_pow<6>(ia_params->LJ_sig / r_off);
+  if ((dist < ia_params->lj.cut + ia_params->lj.offset) &&
+      (dist > ia_params->lj.min + ia_params->lj.offset)) {
+    auto const r_off = dist - ia_params->lj.offset;
+    auto const frac6 = Utils::int_pow<6>(ia_params->lj.sig / r_off);
     auto const fac =
-        48.0 * ia_params->LJ_eps * frac6 * (frac6 - 0.5) / (r_off * dist);
+        48.0 * ia_params->lj.eps * frac6 * (frac6 - 0.5) / (r_off * dist);
```

Comment:
in which case is the new parameter substruct name in uppercase/lowercase/capitalized? LJ becomes lj and MORSE becomes morse, yet GB remains GB and BUCK remains BUCK

Reply:
i will change it for `lj` since i just missed it, for the others e.g. MORSE there is a macro with the same name

Reply:
since these non-bonded IAs objects are members of a struct, you could have all of them in lowercase, similar to the bonded IAs (`union Bond_parameters`, where even the `IBM` acronym is in lowercase), i.e.:
```c++
struct IA_parameters {
...
#ifdef LENNARD_JONES
  LJ_Parameters lj;
#endif
#ifdef WCA
  WCA_Parameters wca;
#endif
#ifdef BMHTF_NACL
  BMHTF_Parameters bmhtf;
#endif
...
```

Reply:
Please have a look a [our naming conventions](https://github.com/espressomd/espresso/wiki/Naming-conventions-and-folder-structure), we use lower case snake case for objects, including data members. Type should be camel case. Please keep the `_Parameters` for the names of the structs, because almost all of then would collide with macro names otherwise.

Reply:
I also think the member names should be spelled out. `BUCK` and `soft` are not good names.